docs/source/cn/array_ops.py

```diff
@@ -732,3 +732,197 @@
 
     """,
 )
+
+reset_docstr(
+    oneflow.load,
+    r"""加载一个被 oneflow.save() 保存的对象。
+
+    参数：
+        - **path** (str): 对象所在的路径
+        - **consistent_src_rank** (int, optional): 加载相容 (consistent) 张量的需要的秩 (rank)。被指定时，只有秩与 consistent_src_rank相等的进程才会真正读取 `path` 中的文件，并且被加载对象中的张量会与  placement = `flow.placement('cuda', [consistent_src_rank])` 一致。
+
+    返回类型：
+        加载好的对象
+    """
+)
+
+reset_docstr(
+    oneflow.numel,
+    r"""
+    numel(input) -> int
+
+    返回 :attr:`input` 张量中的元素总数量。
+
+    参数：
+        - **input** (oneflow.Tensor): 输入张量
+
+    .. code-block:: python
+
+        >>> import oneflow as flow
+
+        >>> a = flow.randn(1, 2, 3, 4, 5)
+        >>> flow.numel(a)
+        120
+        >>> a = flow.zeros(4,4)
+        >>> flow.numel(a)
+        16
+    """
+)
+
+reset_docstr(
+    oneflow.gather_nd,
+    r"""
+    oneflow.gather_nd(input, index) -> Tensor
+    
+    该算子为 `gather` 的高维扩展， `index` 是一个k维张量，以作为输入张量 `input` 的索引。
```

Comment:
```suggestion
    该算子为 `gather` 的高维扩展， `index` 是一个k维张量，以作为输入张量 `input` 的索引。
```

这里的说法可能不准确，弄清楚原因需要知道几个 gather 算子的关系：
tensorflow 里有 gather 和 gather_nd。也就是 tf.gather, tf.gather_nd。 然后 `tf.gather_nd` 是 `tf.gather` 的高维扩展。

pytorch 里只有  gather，也就是 `torch.gather`。

oneflow 因为历史遗留问题，目前的现状是： `oneflow.gather` 等价于 `torch.gather`； `oneflow.gather_nd` 等价于 `tf.gather_nd`。

所以这里讲 gather_nd 是 gather 的扩展，是不准确的。不如说它等价于 tensorflow.gather_nd（给出tf的文档链接）。

你了解下背景后，看看怎么改准确点吧。

Reply:
这边完全是按照英文翻译翻的 不过我还是把他改成等价于tf.gather_nd吧